packages/lisk-transactions/src/base_transaction.ts

```diff
@@ -138,8 +138,18 @@ export abstract class BaseTransaction {
 		const tx = (typeof rawTransaction === 'object' && rawTransaction !== null
 			? rawTransaction
 			: {}) as Partial<TransactionJSON>;
-		this.amount = new BigNum(tx.amount || '0');
-		this.fee = new BigNum(tx.fee || '0');
+		this.amount = new BigNum(
+			(typeof tx.amount === 'string' && tx.amount !== '') ||
```

Comment:
also check for `new BigNum('notanumber')` as that seems to throw error at least on non-forked original version.